src/consensus/aft/async_execution.h

```diff
@@ -0,0 +1,232 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "raft_types.h"
+
+namespace aft
+{
+  class AbstractExecMsgStore
```

Comment:
I think I'm confused by the name of this class, this is presumably not a store?

Reply:
AbstractConsensusCb